Unity/Assets/YarnSpinner/Editor/TheSecretLab.YarnSpinner.Editor.asmdef

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "TheSecretLab.YarnSpinner.Editor",
```

Comment:
Change name to "YarnSpinner.Editor"

Reply:
Done, but I changed the name to `YarnSpinnerUnity.Editor` instead to be consistent with the naming: ac9b44da31cfc95d01bc33ea284ef9135e09933b